app/src/main/java/net/squanchy/tweets/TweetsPageView.java

```diff
@@ -84,12 +89,26 @@ private void initList() {
         }
 
         emptyView.setText(context.getString(R.string.no_tweets_for_query, query));
+
+        LinearLayoutManager layoutManager = (LinearLayoutManager) tweetsList.getLayoutManager();
+
+        scrollListener = new ScrollListener(layoutManager) {
```

Comment:
Can we have the scroll listener accept a `LoadMoreListener` which has only that method instead of extending the class itself?